baseplate/lib/prometheus_metrics.py

```diff
@@ -36,3 +41,19 @@ def getHTTPSuccessLabel(httpStatusCode: int) -> str:
     The HTTP success label is "true" if the status code is 2xx or 3xx, "false" otherwise.
     """
     return str(200 <= httpStatusCode < 400).lower()
+
+
+def are_metrics_enabled(raw_config: Dict[str, str]) -> bool:
```

Comment:
generally, despite the bad grammar, the pattern is to prefix it with `is` instead of `are`